pyproject.toml

```diff
@@ -50,6 +50,7 @@ mysql = ["mysql-connector-python"]
 pgsql = ["psycopg2"]
```

Comment:
Should we call this extra `postgres` instad? `pgsql` isn't even the name of the package 😅

Reply:
I think it was to stay in the same format as mysql/mssql.

Anyway, I'm fine with either one.

Reply:
Let's switch to `postgres` then